controller.py

```diff
@@ -1,7 +1,8 @@
 import base64 as stdlib_base64
 from nose.tools import set_trace
 from datetime import datetime
-from flask import request, make_response
+import flask
```

Comment:
I'm importing flask directly so we can refer to `flask.request` (an established pattern we use to mean "the current request") rather than just `request`.